src/app/container/register-tool/register-tool.service.ts

```diff
@@ -28,7 +28,9 @@ import { HostedService } from './../../shared/swagger/api/hosted.service';
 import { MetadataService } from './../../shared/swagger/api/metadata.service';
 import { DockstoreTool } from './../../shared/swagger/model/dockstoreTool';
 import { Tool } from './tool';
-import { finalize } from 'rxjs/operators';
+
+// This line is super important for jQuery to work across the website for some reason
+import * as $ from 'jquery';
```

Comment:
What's going on here? (maybe a link to an issue)

Reply:
just rearranging the imports in this PR.  Previously added to get jquery to work (not by me)

Reply:
See if there is an issue or something to describe the issue nonetheless, this looks odd

Reply:
https://github.com/ga4gh/dockstore/issues/1763